package-lock.json

```diff
@@ -18,6 +18,54 @@
         "typescript": "^5.2.2"
       }
     },
+    "../../../../../nix/store/4bgg8j8275adawqbc5rkm4fv48dgg2i3-nodejs-18.18.2/lib/node_modules/corepack": {
+      "version": "0.19.0",
```

Comment:
@jaredponn be warned, this is probably not what you wanted

Reply:
Wow, thanks so much for catching this and letting me know. I certainly did not intend this!